lms/app.py

```diff
@@ -100,6 +100,9 @@ def create_app(global_config, **settings):  # pylint: disable=unused-argument
     config.include("lms.tweens")
     config.add_static_view(name="export", path="lms:static/export")
     config.add_static_view(name="static", path="lms:static")
+    # For ui-playground only
```

Comment:
I needed to expose these two static files which get built to this folder, is this the appropriate way to do it?

Reply:
Will do